apps/src/music/player/MusicLibrary.ts

```diff
@@ -437,17 +443,22 @@ export interface SoundFolder {
   skipLocalization?: boolean;
 }
 
+interface SoundFolderInstrument extends Omit<SoundFolder, 'sounds'> {
```

Comment:
`InstrumentData`/`KitData`? I know all these terms are fairly overloaded in our codebase though, so also fine with what you have